tests/js/spec/components/dataExport.spec.jsx

```diff
@@ -79,10 +79,13 @@ describe('DataExport', function() {
       <WrappedDataExport payload={mockPayload} />,
       mockRouterContext(mockAuthorizedOrg)
     );
-    wrapper.find('button').simulate('click');
     expect(wrapper.find(DataExport).state()).toEqual({
       inProgress: false,
     });
+    wrapper.find('button').simulate('click');
+    expect(wrapper.find(DataExport).state()).toEqual({
+      inProgress: true,
+    });
```

Comment:
I would avoid coupling tests to internal component state. Instead could you look at attributes of the button and assert that the button was disabled?